tests/dragonfly/generic_test.py

```diff
@@ -0,0 +1,12 @@
+from dragonfly import dfly_multi_test_args
+
+
+@dfly_multi_test_args({'keys_output_limit': 512}, {'keys_output_limit': 1024})
+class TestKeys:
+    def test_max_keys(self, client, df_server):
+        max_keys = df_server['keys_output_limit']
+
+        for x in range(max_keys+10):
+            client.set(str(x), str(x))
+        keys = client.keys()
+        assert len(keys) == max_keys
```

Comment:
This test was wrong, so I fixed it. But the funny thing is that `keys_output_limit` seems not to work on df side 😅